x-pack/plugins/observability_solution/apm/server/routes/services/route.ts

```diff
@@ -495,7 +496,7 @@ const serviceThroughputRoute = createApmServerRoute({
     }),
     query: t.intersection([
       t.type({ transactionType: t.string, bucketSizeInSeconds: toNumberRt }),
-      t.partial({ transactionName: t.string }),
+      t.partial({ transactionName: t.string, filters: t.string }),
```

Comment:
What do you think of creating a custom io-ts type that validates and parses the string to a valid Filter type? Something similar to what you already [here](https://github.com/elastic/kibana/pull/181359/files#diff-49d1ccb0a625c6cab1a2aab22bff950101d83bd0aeb28335a143b0ca822e950fR10).